index.js

```diff
@@ -68,3 +69,16 @@ function sortByLast(a, b) {
   }
   return 0
 }
+
+function search() {
+  $("#search").on("keyup", function(event) {
+    var text = $("#search").val().toLowerCase()
+    $(".name").each(function(index) {
+      if ($(this).text().toLowerCase().includes(text)) {
+        $(this).closest("div").show()
```

Comment:
`closest()` is a tricky function. It works and is totally legitimate and fine. It becomes fragile if something changes. If you (or someone else) adds a `div` and becomes the "closest" div, this breaks. A more stable way of doing this would be to use `id`s as identifiers. People are much less likely to change an id than they are to add a `div`.